Quaver.Shared/Screens/Gameplay/Rulesets/Keys/GameplayRulesetKeys.cs

```diff
@@ -30,23 +32,26 @@ public class GameplayRulesetKeys : GameplayRuleset
         ///     the playfield and the HitObjectManager.
         ///
         ///     We can't intiialize it in Playfield as that gets created first.
+        ///
+        ///     This is a list because multiple scroll directions require multiple Timing Line Managers.
+        /// 
         /// </summary>
-        public TimingLineManager TimingLineManager { get; }
+        public List<TimingLineManager> TimingLineManager { get; } = new List<TimingLineManager>();
```

Comment:
The timing lines on 7k split scroll are broken

![img](https://i.imgur.com/w5UB4wL.png)